website/source/docs/job-specification/template.html.md

```diff
@@ -121,5 +121,13 @@ template {
 }
 ```
 
+### Client Configuration
+
+The `template` block has the following [client configuration
+options](/docs/agent/config.html#options):
+
+* `template.allow_host_source` - Allows templates to specify their source
```

Comment:
wouldn't it be safer to be default `off` so people need to opt into "insecure" mode?